### ISSUE TITLE
It looks like open() logs a registry key on failure

### ISSUE THREAD

**Issue by Pomax** [EvenAR/node-simconnect#68]
When running the following code, the library seems to log the MSFS Windows registry key to the console:

```javascript
import { open, Protocol } from "node-simconnect";
try {
    await open("My app", Protocol.KittyHawk);
} catch (e) {
    // ignore the error
}
```

Logs:

```
D:\temp>node test.js
{"HKCU\\Software\\Microsoft\\Microsoft Games\\Flight Simulator": "

D:\temp>
```

When MSFS isn't running, or loading in.

When MSFS is fully loaded, that console log does not occur.

**Comment by EvenAR** [comment]
Weird. Looks like bumping the regedit dependency solved it. Thanks for the issue!

**Comment by EvenAR** [comment]
> Silly request, but could that update be published as v3.4.1? =D
> 
> (that way I can bump the api wrapper package, too)

New version published :)

**Comment by Pomax** [comment]
❤️
